### ISSUE TITLE
64bit Windows release

### ISSUE THREAD

**Issue by Stanzilla** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2146]
Any plans to add a 64bit client release for Windows?

**Comment by MarshallOfSound** [comment]
There is currently no plan to add a 64 bit release for windows.  Why you might ask? 😆 

* It adds another point of failure for the application
* Adds another architecture to test
* The update server I use doesn't support it very well
* 32bit applications are smaller and run on both 32bit and 64bit
* We still support windows 7 (which could be 32bit), when Chromium drops windows 7 support, which won't be for many years, we can consider dropping 32bit support.

TLDR: A long way in the future it might happen, but there is no real reason for it to happen right now
